src/plugins/discover/public/application/embeddable/saved_search_embeddable.tsx

```diff
@@ -6,67 +6,55 @@
  * Side Public License, v 1.
  */
 
-import './search_embeddable.scss';
-import angular from 'angular';
-import _ from 'lodash';
 import { Subscription } from 'rxjs';
+import React from 'react';
+import ReactDOM from 'react-dom';
 import { i18n } from '@kbn/i18n';
-import { UiActionsStart } from '../../../../ui_actions/public';
-import { RequestAdapter, Adapters } from '../../../../inspector/public';
-import {
-  APPLY_FILTER_TRIGGER,
-  esFilters,
-  Filter,
-  TimeRange,
-  FilterManager,
-  Query,
-  IFieldType,
-} from '../../../../data/public';
+import _ from 'lodash';
```

Comment:
good occasion to get rid of importing whole of lodash, `isEqual` should be sufficient